### PR TITLE
LOG-5381: add alerts metric and dashboard panels

### PR DESCRIPTION
**Pull request by vparfonov**
Signed-off-by: Vitalii Parfonov <vparfono@redhat.com>
### Description
This PR introduces monitoring and alerting for Vector collector output buffer size.

#### Alert: 
Added alerts to notify when Vector output buffer disk usage reaches a critical threshold.
buffer-alert

####  Metrics Dashboards (2 panels):
Panel 1: Shows the absolute size of the Vector output buffer, summed by hostname, namespace, and CLF instance name 
buffer-metric-1
Panel 2: Shows the percentage of buffer size relative to the total disk space available on the node.
buffer-metric-2

/cc 
/assign 

/cherry-pick 

### Links

- Depending on PR(s):
- Bugzilla:
- Github issue:
- JIRA:
- Enhancement proposal:

### CODE REVIEW

File: config/prometheus/collector_alerts.yaml [hunk 1]
```diff
@@ -85,6 +85,17 @@ spec:
       labels:
         service: visualization
         severity: Warning
+    - alert: DiskBufferUsage
+      annotations:
+        message: "Collectors potentially consuming too much node disk, {{ $value }}% "
+        summary: "Detected consuming too much node disk on $labels.hostname host"
```
Comment (jcantrill) [inline config/prometheus/collector_alerts.yaml:91]: Does this need curly brackets?  The sample alert in the PR shows it as-is here

File: docs/administration/collector-metrics-and-alerts.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
+= Collector Metrics and Alerts
```
Comment (jcantrill) [inline docs/administration/collector-metrics-and-alerts.adoc:1]: We may need to add a note here this only applies to "infra" namespaces and we have no details for collectors deployed to non-infra NS.  Unless you were able to learn if it is possible to collect metrics from non-infra namespaces by some enablement

**Review by jcantrill — APPROVED** [review]

File: docs/administration/collector-metrics-and-alerts.adoc (hunk 2 above)
Comment (vparfonov) [reply docs/administration/collector-metrics-and-alerts.adoc:1]: added in docs how to enable

**Pull request merged**